flexmeasures/data/schemas/reporting/aggregation.py

```diff
@@ -0,0 +1,65 @@
+from marshmallow import fields, ValidationError, validates_schema
+
+from flexmeasures.data.schemas.reporting import ReporterConfigSchema
+
+from enum import Enum
+
+
+class AggregationMethod(Enum):
+    SUM = "SUM"
+    MEAN = "MEAN"
```

Comment:
Lowercase would make them more in sync with Pandas method names.
```
SUM = "sum"
MEAN = "mean"
```

This would also allow you to simplify lines 66-69 in `flexmeasures/data/models/reporting/aggregatator.py`, also making it easier to add support for other methods, simply by extending the `AggregationMethod` class, for example, by adding `MAX = "max"`.

You could also lowercase the keys (so they also become lowercased in the example, and will feel more like Pandas method names there). But maybe there is a reason why you capitalized these (and the aliases) in the first place? Maybe just so they become emphasized in the config?

Reply:
Agreed!

The capitalization in the alias is to highlight them.